common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -3658,6 +3658,19 @@ static jint evp_aead_ctx_op_buf(JNIEnv* env, jlong evpAeadRef, jbyteArray keyArr
     inBuf += in_position;
     outBuf += out_position;
 
+    size_t inSize = in_limit - in_position;
+    uint8_t* outBufEnd = outBuf + out_limit - out_position;
+    uint8_t* inBufEnd = inBuf + inSize;
+    std::unique_ptr<uint8_t[]> inCopy(new(std::nothrow) uint8_t[inSize]);
```

Comment:
I think we discussed this in chat, maybe you're missing a commit here?  You can leave `inCopy` uninitialised here and then use `inCopy.reset(new(std::nothrow) uint8_t[inSize])` in the if branch if needed.

Reply:
My bad I forgot to push. It is initialized inside now thank you!